### PR TITLE
bump monorepo

### PR DESCRIPTION
**Pull request by alexfauquette**
- [ ] I have followed (at least) the [PR section of the contributing guide](https://github.com/mui/mui-x/blob/HEAD/CONTRIBUTING.md#sending-a-pull-request).

### CODE REVIEW

**Comment by flaviendelangle** [comment]
@alexfauquette if you can update to include https://github.com/mui/material-ui/pull/41033 :pray:

File: yarn.lock [hunk 1]
```diff
@@ -6549,56 +6552,70 @@ error-ex@^1.3.1:
...
-  version "1.22.3"
-  resolved "https://registry.yarnpkg.com/es-abstract/-/es-abstract-1.22.3.tgz#48e79f5573198de6dee3589195727f4f74bc4f32"
-  integrity sha512-eiiY8HQeYfYH2Con2berK+To6GrK2RxbPawDkGq4UiCQQfZHb6wX9qQqkbpPqaxQFcl8d9QzZqo0tGE0VcrdwA==
-  dependencies:
-    array-buffer-byte-length "^1.0.0"
-    arraybuffer.prototype.slice "^1.0.2"
-    available-typed-arrays "^1.0.5"
-    call-bind "^1.0.5"
-    es-set-tostringtag "^2.0.1"
+es-abstract@^1.22.1, es-abstract@^1.22.3:
```
Comment (LukasTy) [inline yarn.lock:6555]: Nitpick: Are all these additional bumps related to the monorepo bump? 🤔 
At first sight, it seems like a "job" for "lock file maintenance". 🙈 🤷

**Comment by LukasTy** [comment]
> if you can update to include [mui/material-ui#41033](https://github.com/mui/material-ui/pull/41033) 🙏

Does it matter if it is included in the monorepo? 🤔 
It is a separately published package as far as I can see. 🤷

File: yarn.lock (hunk 1 above)
Comment (alexfauquette) [reply yarn.lock:6555]: I just run `yarn upgrade @mui/monorepo` and `yarn deduplicate`

Comment (LukasTy) [reply yarn.lock:6555]: 🙈 
Have you tried only updating what's necessary?
Try deleting the `monorepo` entry in the lock file and running `yarn`. 🤷

**Comment by flaviendelangle** [comment]
Oh right I forgot... :p

File: yarn.lock (hunk 1 above)
Comment (alexfauquette) [reply yarn.lock]: Seems to work 🙈🤯

Comment (LukasTy) [reply yarn.lock]: Nice. 👏

**Review by LukasTy — APPROVED** [review]

**Pull request merged**